hashes/src/hmac/mod.rs

```diff
@@ -35,6 +35,30 @@ impl<T: GeneralHash> Hmac<T> {
     {
         HmacEngine::new(key)
     }
+
+    /// Hashes some bytes.
+    pub fn hash(key: &[u8], data: &[u8]) -> Self
```

Comment:
Damn, this sucks, but I don't see any sensible way of fixing it. These are the times when I wish for Swift-like named arguments.